src/Header/HeaderContent.tsx

```diff
@@ -11,18 +11,21 @@ export interface HeaderContentProps extends HTMLAttributes<HTMLElement> {
   megamenu?: boolean;
   /** Parametro per il controllo responsive del componente secondo Bootstrap Italia. */
   expand?: boolean | 'sm' | 'md' | 'lg' | 'xl';
+  /** Parameter for small/narrow variant of header content **/
+  small?: boolean;
 }
 
 export class HeaderContent extends PureComponent<HeaderContentProps> {
   static contextType = HeaderContext;
   render() {
-    const { className, megamenu, ...attributes } = this.props;
+    const { className, megamenu, small, ...attributes } = this.props;
     const { type } = this.context;
     const classes = classNames(className, {
       'it-header-slim-wrapper-content': type === SLIM,
       'it-header-center-content-wrapper': type === CENTER,
       navbar: type === NAVBAR,
-      'has-megamenu': megamenu
+      'has-megamenu': megamenu,
+      'it-small-header': small
```

Comment:
I do not think this has to go in here. This class belongs to the wrapper component `Header`, not to the content component.
Look here for the class `it-small-header`: https://italia.github.io/bootstrap-italia/docs/menu-di-navigazione/header/#versione-stretta

Reply:
Alright got it, I guess I'll have to send `type` as Center and `isSmall` true to achieve what's in the BI. Thanks